source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
-   following command:
+1. To move to the top level of the website’s root directory, enter thefollowing command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       cd /var/www/html
 
 2. To download the latest version of WordPress, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo curl -O https://wordpress.org/latest.tar.gz
 
-3. To extract all the files and directories from the downloaded file,
-   enter the following command:
+3. To extract all files and directories from the downloaded file, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo tar –xzvf latest.tar.gz
 
-4. To rename the top-level WordPress directory to “blog”, enter the following
-   command:
+4. To rename the top-level WordPress directory to “blog,” enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo mv wordpress blog
 
 5. To remove the downloaded tar file, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo rm latest.tar.gz
 
-Setting up WordPress
+Set up WordPress
 --------------------
 
-With the WordPress components loaded into the
-:file:`/var/www/html/blog` directory, we can set everything up.
+Recall that you created a database and user when you installed phpMyAdmin during :ref:`Creating a Clear Linux based web server <web-server-install>`. To continue with the WordPress setup, go to: http://<your_website>/blog/wp-admin/install.php.
 
-Instead of editing the :file:`wp_config.php` file manually, we are using the
-web-based configuration tool to setup the database name and user. We created
-both in the :ref:`web server tutorial <web-server-install>`. In your browser,
-go to: http://<your_website>/blog/wp-admin/install.php.
-
-Your screen should look like figure 1:
+The WordPress language option screen appears, as shown in figure 1. Select :guilabel:`English` and click :guilabel:`Continue`.
 
 .. _fig-wp-install-1:
 
 .. figure:: figures/wp-install-1.png
     :alt: WordPress language selection
     :width:     600
 
-    WordPress language selection screen.
-
-Select :guilabel:`English` for the language and click the
-:guilabel:`Continue` button. :ref:`As shown on figure 1<fig-wp-install-1>`
+    Figure 1: WordPress language selection screen.
 
 
-
-The WordPress installation continues until the welcome screen shown in figure
-2 appears:
+The WordPress installation continues until the Welcome screen appears, as shown in figure 2:
 
 .. figure:: figures/wp-install-2.png
     :alt: WordPress welcome screen
     :width:     600
 
-    WordPress welcome screen.
-
-Click the :guilabel:`Let’s go!` button to enter the information.
+    Figure 2: WordPress welcome screen.
 
-Enter the database name, username, and password we used when creating the
-database:
+Click :guilabel:`Let’s go!`. Enter pertinent information in the screen that appears as shown in Figure 3 below.
```

Comment:
Revise to: 
#. Click :guilabel:`Let’s go!`.
#. Enter the database name, username, and password we used when creating the database.

Reply:
done